components/ItemCard/ItemCard.js

```diff
@@ -0,0 +1,88 @@
+import {
+  StyledListItem,
+  StyledArticle,
+  StyledDiv,
+  StyledH2,
+  StyledParagraph,
+  StyledThumbnail,
+  WrapperDiv,
+  StyledForm,
+  StyledInput,
+  StyledButton,
+  StyledSVG,
+} from "@/components/ItemCard/ItemCard.styled.js";
+
+export function ItemCard({ product }) {
+  return (
+    <StyledListItem>
+      <StyledArticle>
+        <StyledThumbnail
+          src={product.thumbnail}
+          alt={product.title}
+          width={120}
+          height={152}
+        />
+        <StyledDiv>
+          <StyledH2>{product.title}</StyledH2>
+          <StyledParagraph>{product.price}€</StyledParagraph>
+          <WrapperDiv>
```

Comment:
What is this div doing, other than wrapping your form? Are you sure you need it?

Reply:
- [x] Removed unused WrapperDiv styled component